elm-src/main.elm

```diff
@@ -69,6 +71,14 @@ type SuperString = Super String
 update : Msg -> Model -> (Model, Cmd Msg)
 update msg model =
   case msg of
+    Posted _ ->
+        (Loading, 
+            Http.get
+            {
+                url = "/feed",
```

Comment:
Can we turn these endpoints to constants somewhere in a root scope? I'm about to change them.